Dockerfile

```diff
@@ -1,20 +1,13 @@
-FROM golang:alpine as build
-RUN apk --update --no-cache add ca-certificates git make
-ENV CGO_ENABLED=0
```

Comment:
If you meant to enable cgo, distroless/base would be the way to go instead of distroless/static.